gobblin-kubernetes/gobblin-service/base-cluster/deployment.yaml

```diff
@@ -61,19 +60,4 @@ spec:
           image: will97/gobblin-standalone:latest
           volumeMounts:
             - name: shared-jobs
-              mountPath: /tmp/gobblin-standalone/jobs
----
-apiVersion: v1
```

Comment:
We don't need it?

Reply:
Huh I'm not sure why git says I removed the line, I only removed the newline after it because it shows up on the other side of the diff. Probably the lack of a newline? The volume is still there.
I removed the service underneath it to move it to a new file to better separate the yamls